instrumentation/apache-log4j-1/src/test/java/org/apache/log4j/Category_InstrumentationTest.java

```diff
@@ -1,16 +1,21 @@
-package com.nr.agent.instrumentation.log4j1;
+/*
+ *
+ *  * Copyright 2022 New Relic Corporation. All rights reserved.
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package org.apache.log4j;
 
 import com.newrelic.agent.introspec.InstrumentationTestConfig;
 import com.newrelic.agent.introspec.InstrumentationTestRunner;
 import com.newrelic.agent.introspec.MetricsHelper;
-import org.apache.log4j.Level;
-import org.apache.log4j.Logger;
 import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
 @RunWith(InstrumentationTestRunner.class)
-@InstrumentationTestConfig(includePrefixes = { "org.apache.log4j" }, configName = "application_logging_enabled.yml")
+@InstrumentationTestConfig(includePrefixes = {"org.apache.log4j"}, configName = "application_logging_enabled.yml")
 public class Category_InstrumentationTest {
```

Comment:
I don't see any tests in here that actually verify that the LogEvents are being created at the various log levels. 

Is there a reason this can't be done like it is for all of the other logging framework instrumentation modules? Here's the log4j2 test for example:

https://github.com/newrelic/newrelic-java-agent/blob/7e7c4c6575ef800ef24c941d4e13cefe269e8a13/instrumentation/apache-log4j-2/src/test/java/com/nr/agent/instrumentation/log4j2/LoggerConfig_InstrumentationTest.java#L46-L440

Reply:
Man I wish our InstrumentationTestRunners were JUnit5 compatible.  Nice tests though, so I'm stealing mostly.